src/components/SignupForm.js

```diff
@@ -21,25 +23,34 @@ const signupSchema = Yup.object().shape({
     .required('**Password is required'),
 });
 
-const SignupForm = () => {
+const SignupForm = (props) => {
```

Comment:
What props is `SignupForm` taking in? Honest question, not trying to be snarky or anything. If it is taking in any props, the linter needs prop type validation (see [https://github.com/Lambda-School-Labs/life-logger-fe/tree/development#a-note-on-eslint](https://github.com/Lambda-School-Labs/life-logger-fe/tree/development#a-note-on-eslint))

Reply:
yes we needs props here to get history . do we needs too specify prop types to merge this in ? all the other components pulled in yesterday are not doing this ?